README.md

```diff
@@ -1,236 +1,56 @@
-
-<a href="https://github.com/viromedia/viro/tree/master/code-samples/js/HumanBody">
-<img src="https://raw.githubusercontent.com/viromedia/viro/master/code-samples/js/HumanBody/heart_demo.gif">
-</a>
-
-This example showcases 3d objects. Orbit around a 3d Heart to see it from different angles!
-
-### [VR MediaPlayer](https://github.com/viromedia/viro/tree/master/code-samples/js/ViroMediaPlayer)
-
-<a href="https://github.com/viromedia/viro/tree/master/code-samples/js/ViroMediaPlayer">
-<img src="https://raw.githubusercontent.com/viromedia/viro/master/code-samples/js/ViroMediaPlayer/movie_theater.gif">
-</a>
-
-Learn how to display and play 2d and 360 video with interactive play controls that can play, pause and stop.
-
-### [Product Showcase](https://github.com/viromedia/viro/tree/master/code-samples/js/ProductShowcase)
-
-<a href="https://github.com/viromedia/viro/tree/master/code-samples/js/ProductShowcase">
-<img src="https://raw.githubusercontent.com/viromedia/viro/master/code-samples/js/ProductShowcase/product_showcase.gif">
-</a>
+<p align="center" style="background-color: #CCCCCC;">
+  <a href="https://virocommunity.github.io/">
+    <img src="https://raw.githubusercontent.com/ViroCommunity/virocommunity.github.io/main/viro_community_logo.png" alt="Viro Community logo">
+  </a>
+</p>
 
-Learn how to display and play 2d and 360 video with interactive play controls that can play, pause and stop.
-A demonstration on how to do an interactive shopping app for TV's. Uses flexbox for UI and 3d objects with animations.
+[![npm version](https://img.shields.io/npm/v/@viro-community/react-viro)](https://www.npmjs.com/package/@viro-community/react-viro)
+[![downloads](https://img.shields.io/npm/dm/@viro-community/react-viro?color=purple)](https://www.npmjs.com/package/@viro-community/react-viro)
+[![Discord](https://img.shields.io/discord/774471080713781259?label=Discord)](https://discord.gg/YfxDBGTxvG)
 
-### [AR Sample](https://github.com/viromedia/viro/tree/master/code-samples/js/ARSample)
-
-<a href="https://github.com/viromedia/viro/tree/master/code-samples/js/ARSample">
-<img src="https://raw.githubusercontent.com/viromedia/viro/master/code-samples/js/ARSample/ar_sample.gif">
-</a>
-
-A scene with objects, text and animation displayed on ARKit planes detected in the scene.
-
-### [AR Car Marker Demo](https://github.com/viromedia/viro/tree/master/code-samples/js/ARCarDemo)
-
-<a href="https://github.com/viromedia/viro/tree/master/code-samples/js/ARCarDemo">
-<img src="https://raw.githubusercontent.com/viromedia/viro/master/code-samples/js/ARCarDemo/viro_car_marker_demo.gif">
-</a>
-
-An ARScene that looks for this [image](https://github.com/viromedia/viro/tree/master/code-samples/js/ARCarDemo/res/logo.png) and adds a model of a car with the ability to change its colors.
-
-### [AR Driving Car Demo](https://github.com/viromedia/viro/tree/master/js/ARDrivingCarDemo)
+# Viro React
```

Comment:
We settled up that the package should be called ViroReact :D
![image](https://user-images.githubusercontent.com/33497067/111711145-626a7200-884b-11eb-815c-a609ef7e1e8c.png)